cli/cli_fileshare.go

```diff
@@ -146,7 +128,7 @@ func (c *cmd) IsFileshareDaemonReachable(ctx *cli.Context) error {
 		return formatError(fmt.Errorf(MsgMeshnetNotEnabled))
 	}
 
-	return startupErrorToErrorMessage(c.fileshareProcessManager.StartProcess())
+	return nil
```

Comment:
How it the filesharing starting initiated from CLI now?

Reply:
Fileshare is now always started by norduser, which can be started on any user command(on daemon side at this moment, but will also be started on cli side for snap purposes in future PRs). When meshnet is enabled, daemon will tell norduser to startup fileshare. In future PRs, norduser also queries the main daemon for meshent on startup and enables fileshare automatically.